pixi.toml

```diff
@@ -471,13 +483,19 @@ pip = ">=23"
 [feature.python-pypi]
 platforms = ["linux-64", "osx-arm64", "osx-64", "win-64"]
 
+[feature.python-pypi.dependencies]
+# Note these are the same as in `python-dev` but we need to repeat them here because the two are mutually exclusive.
+opencv = ">4.6"
+numpy = "<2"    # Rerun still needs numpy <2. Enforce this outside of the pypi dep tree so we pick up the conda version.
+
+
 [feature.python-pypi.pypi-dependencies]
 rerun-sdk = "==0.17.0"
 rerun-notebook = "==0.17.0"
 
 # EXAMPLES ENVIRONMENT
 [feature.examples-common]
-platforms = ["linux-64", "osx-arm64", "osx-64", "win-64"]
+platforms = ["linux-64"]
```

Comment:
That will break the ability to run example on mac I believe.

Reply:
Indeed:

```
❯ pixi run -e examples py-build-examples && pixi run -e examples detect_and_track_objects
  × The current platform does not satisfy the minimal virtual package requirements
  ╰─▶ the environment 'examples' does not support 'osx-arm64'
```

Reply:
Didn't mean to submit that. Was speeding up some testing cycles.